### PR TITLE
support mountOptions in configmap

### PR DESCRIPTION
**Pull request by zxh326**

### CODE REVIEW

File: pkg/config/setting.go [hunk 1]
```diff
@@ -728,6 +732,19 @@ func applyAttrPatch(attr *PodAttr, setting *JfsSetting) {
 	attr.VolumeMounts = patch.VolumeMounts
 	attr.Volumes = patch.Volumes
 	attr.Env = patch.Env
+
+	// merge options
+	if len(setting.Options) > 0 && len(patch.MountOptions) > 0 {
```
Comment (zwwhdls) [inline pkg/config/setting.go]: what if len(setting.Options) == 0 and len(patch.MountOpions) > 0 ?

Comment (zxh326) [reply pkg/config/setting.go]: fixed

File: pkg/config/setting.go [hunk 2]
```diff
@@ -728,6 +732,19 @@ func applyAttrPatch(attr *PodAttr, setting *JfsSetting) {
 	attr.VolumeMounts = patch.VolumeMounts
 	attr.Volumes = patch.Volumes
 	attr.Env = patch.Env
+
+	// merge or overwrite setting options
+	if setting.Options == nil {
+		setting.Options = make([]string, 0)
+	}
+	for _, option := range patch.MountOptions {
```
Comment (zwwhdls) [inline pkg/config/setting.go:740]: if patch.MountOptions is nil?

Comment (zxh326) [reply pkg/config/setting.go:740]: for loop will not execute

**Review by zwwhdls — APPROVED** [review]

**Pull request merged**